metriq-api/api-routes.js

```diff
@@ -18,6 +18,9 @@ const tagController = require('./controller/tagController')
 const taskController = require('./controller/taskController')
 const resultController = require('./controller/resultController')
 const methodController = require('./controller/methodController')
+const platformController = require('./controller/platformController')
+const propertyController = require('./controller/propertyController')
+const dataTypeController = require('./controller/dataTypeController')
```

Comment:
Minor comment. Do we want to alphabetize these imports? Or do the orders of these matter?

Reply:
Might as well, simply to have a standard. (The _route_ declaration behavior below is semi-dependent on order, but the `require(...)` order shouldn't matter, unless something is wrong.)